src/DetectorConstruction.cc

```diff
@@ -98,9 +102,59 @@ void DetectorConstruction::LoadGeometryForRun(G4int nr)
 
 }
 
+//JC
+
+void DetectorConstruction::ConstructTargetRun7()
+{
+	G4RotationMatrix rot = G4RotationMatrix();
+
+   G4double z[] = {-8.8*cm, -7.8*cm, -7.8*cm, -6.6*cm, -6.6*cm, -5.1*cm, -4.9*cm, -3.1*cm, -2.8*cm, 2.8*cm, 3.1*cm, 4.9*cm, 5.1*cm, 6.6*cm, 6.6*cm, 7.8*cm, 7.8*cm, 8.8*cm }; 
+   G4double rInner[] = { 6.0*mm, 6.0*mm, 6.0*mm, 5.0*mm, 5.0*mm, 5.0*mm, 5.0*mm , 5.0*mm,  5.0*mm, 5.0*mm, 5.0*mm, 5.0*mm, 5.0*mm, 5.0*mm, 5.0*mm, 6.0*mm, 6.0*mm , 6.0*mm};
+   G4double rOuter[] = { 17.5*mm, 17.5*mm, 13.5*mm, 13.5*mm,  10.0*mm, 10.0*mm, 10.0*mm, 10.0*mm, 5.0*mm, 5.0*mm, 10.0*mm, 10.0*mm, 10.0*mm, 10.0*mm, 13.5*mm, 13.5*mm, 17.5*mm, 17.5*mm }; 
+
+   G4Polycone* smallChamber = new G4Polycone("bigChamber",0*degree,360*degree, 18 , z, rInner, rOuter);
```

Comment:
bigChamber name is misleading here

Reply:
also while running code a following exception occurs:

-------- WWWW ------- G4Exception-START -------- WWWW -------
*** G4Exception : GeomMgt1002
      issued by : G4Polycone::CalculateExtent()
Triangulation of RZ contour has failed for solid: bigChamber !
Extent has been calculated using boundary box
*** This is just a warning message. ***
-------- WWWW -------- G4Exception-END --------- WWWW -------

please check if size of rOuter and rInner ... are as they suppose to be